tests/cephfs/cephfs_utils.py

```diff
@@ -95,14 +95,31 @@ def get_clients(self, build):
         for client in self.clients:
             node = client.node
             if node.pkg_type == 'rpm':
-                out, rc = node.exec_command(cmd='sudo rpm -qa | grep -w attr')
+                out, rc = node.exec_command(cmd="sudo rpm -qa | grep -w 'attr\|xattr'")
                 output = out.read().decode()
                 output.split()
                 if 'attr' not in output:
                     node.exec_command(cmd='sudo yum install -y attr')
                 node.exec_command(cmd='sudo yum install -y gcc python3-devel',
                                   check_ec=False)
 
+                if 'xattr' not in output:
+                    if (build.endswith('7') or build.startswith('3')):
+                        node.exec_command(cmd="sudo subscription-manager repos --enable"
+                                              " rhel-7-server-optional-rpms --enable"
```

Comment:
we are already enabling repos in `install-prereq.py`. any reason why you need it here again.

Reply:
I have tried using just install-prereq.py , but these repos are not getting enabled